service/controller/resource/deployment/deployment.go

```diff
@@ -77,3 +88,39 @@ func getVPNSubnet(customObject providerv1alpha1.AzureConfig) (*net.IPNet, error)
 
 	return &subnets.VPN, nil
 }
+
+func (r *Resource) initialProvisioning(ctx context.Context, customObject providerv1alpha1.AzureConfig) (string, error) {
```

Comment:
forgive me father for i have sinned

Reply:
`bool` wasn't good enough for you sinner here? :smile:

Reply:
We were already using that parameter, so I didn't want to change it :dancers: